### ISSUE TITLE
Crash on block 0

### ISSUE THREAD

**Issue by wemeetagain** [ChainSafe/lodestar#1046]
Started syncing witti from scratch, w genesis state, shut down node before any peers found.
Re-start and found this error, which crashed and exited:
```
2020-06-24 20:14:11  [CHAIN]            info: Found finalized state at slot 0, starting chain from there 
2020-06-24 20:14:11  [NODE]             info: Starting eth2 beacon node - LODESTAR! 
2020-06-24 20:14:11  [METRICS]          info: Starting metrics HTTP server on port 5000 
2020-06-24 20:14:11  [DB]               info: Connected to LevelDB database at .witti/beacon/chain-db 
2020-06-24 20:14:12  [CHAIN]            info: Chain started, waiting blocks and attestations 
2020-06-24 20:14:12  [CHAIN]            info: Found last known finalized state at epoch #0 root 0x0000000000000000000000000000000000000000000000000000000000000000 
2020-06-24 20:14:12  [CHAIN]            info: Found 1 nonfinalized blocks in database from slot 0 to 0 
AssertionError: Expect false == true
    at assert (/home/cayman/Code/lodestar/packages/lodestar-utils/lib/assert.js:18:11)
    at ArrayDagLMDGHOST.headNode (/home/cayman/Code/lodestar/packages/lodestar/lib/chain/forkChoice/arrayDag/lmdGhost.js:371:31)
    at ArrayDagLMDGHOST.head (/home/cayman/Code/lodestar/packages/lodestar/lib/chain/forkChoice/arrayDag/lmdGhost.js:363:15)
    at ArrayDagLMDGHOST.headBlockRoot (/home/cayman/Code/lodestar/packages/lodestar/lib/chain/forkChoice/arrayDag/lmdGhost.js:381:23)
    at BeaconChain.restoreHeadState (/home/cayman/Code/lodestar/packages/lodestar/lib/chain/chain.js:289:68)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
    at async BeaconChain.start (/home/cayman/Code/lodestar/packages/lodestar/lib/chain/chain.js:169:5)
    at async BeaconNode.start (/home/cayman/Code/lodestar/packages/lodestar/lib/node/nodejs.js:157:5)
    at async Object.run [as handler] (/home/cayman/Code/lodestar/packages/lodestar-cli/lib/lodecli/cmds/beacon/cmds/run/run.js:77:3)
```

**Comment by 3xtr4t3rr3str14l** [comment]
i'm not sure if the OP error that @wemeetagain posted is still happening, but i'm getting a different error in the same spot:

```(node:13312) [DEP0005] DeprecationWarning: Buffer() is deprecated due to security and usability issues. Please use the Buffer.alloc(), Buffer.allocUnsafe(), or Buffer.from() methods instead.
2020-07-06 18:20:00  [CHAIN]            info: Found finalized state at slot 0, starting chain from there 
2020-07-06 18:20:00  [NODE]             info: Starting eth2 beacon node - LODESTAR! 
2020-07-06 18:20:00  [METRICS]          info: Starting metrics HTTP server on port 5000 
2020-07-06 18:20:00  [DB]               info: Connected to LevelDB database at .altona/beacon/chain-db 
2020-07-06 18:20:00  [CHAIN]            info: Chain started, waiting blocks and attestations 
2020-07-06 18:20:00  [CHAIN]            info: Found last known finalized state at epoch #0 root 0x0000000000000000000000000000000000000000000000000000000000000000 
2020-07-06 18:20:00  [CHAIN]            info: Found 1 nonfinalized blocks in database from slot 0 to 0 
 ✖ Error: Justified checkpoint does not exist
    at Object.true (/home/seraph/work/ChainSafe/lodestar/packages/lodestar-utils/lib/assert.js:16:13)
    at ArrayDagLMDGHOST.headNode (/home/seraph/work/ChainSafe/lodestar/packages/lodestar/lib/chain/forkChoice/arrayDag/lmdGhost.js:373:31)
    at ArrayDagLMDGHOST.head (/home/seraph/work/ChainSafe/lodestar/packages/lodestar/lib/chain/forkChoice/arrayDag/lmdGhost.js:365:15)
    at ArrayDagLMDGHOST.headBlockRoot (/home/seraph/work/ChainSafe/lodestar/packages/lodestar/lib/chain/forkChoice/arrayDag/lmdGhost.js:383:23)
    at BeaconChain.restoreHeadState (/home/seraph/work/ChainSafe/lodestar/packages/lodestar/lib/chain/chain.js:291:68)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
    at async BeaconChain.start (/home/seraph/work/ChainSafe/lodestar/packages/lodestar/lib/chain/chain.js:162:5)
    at async BeaconNode.start (/home/seraph/work/ChainSafe/lodestar/packages/lodestar/lib/node/nodejs.js:157:5)
    at async Object.run [as handler] (/home/seraph/work/ChainSafe/lodestar/packages/lodestar-cli/lib/cmds/beacon/cmds/run/run.js:77:3)

```

**Comment by twoeths** [comment]
should be resolved by #1078
